doc/guides/build/kconfig/preprocessor-functions.rst

```diff
@@ -28,6 +28,7 @@ while the ``*_hex`` version returns a hexadecimal value starting with ``0x``.
 
 .. code-block:: none
 
+   $(dt_has_compat,<compatible string>)
```

Comment:
maybe `dt_compat_exists`?

Reply:
I based on the work started in #35398 and the `dt_has_compat` name comes from there. And to me it seems to suit better, as it is consistent with [`DT_HAS_CHOSEN`](https://github.com/zephyrproject-rtos/zephyr/blob/65a2e4b7600845f6fdef18233554e6a9b0e51d04/include/devicetree.h#L1861) and [`DT_HAS_COMPAT_STATUS_OKAY`](https://github.com/zephyrproject-rtos/zephyr/blob/65a2e4b7600845f6fdef18233554e6a9b0e51d04/include/devicetree.h#L2189). @mbolivar-nordic ?

Reply:
makes sense